src/libxtp/threecenter.cc

```diff
@@ -37,8 +37,13 @@ void TCMatrix_dft::Fill(const AOBasis& auxbasis, const AOBasis& dftbasis) {
     _inv_sqrt = auxAOcoulomb.Pseudo_InvSqrt(1e-8);
     _removedfunctions = auxAOcoulomb.Removedfunctions();
   }
-  _matrix = std::vector<Symmetric_Matrix>(
-      auxbasis.AOBasisSize(), Symmetric_Matrix(dftbasis.AOBasisSize()));
+  _matrix = std::vector<Symmetric_Matrix>(auxbasis.AOBasisSize());
+
+#pragma omp parallel for schedule(dynamic, 4)
```

Comment:
this maybe a very naive question, but I still don't get why each thread ask for 4 iterations

Reply:
Actually it is not, I did not want the threads to lock each other up because of false sharing or so, so I thought 4 is a good number. Maybe I am wrong.